test/living-html/htmlelement.js

```diff
@@ -0,0 +1,27 @@
+﻿"use strict";
+
+var jsdom = require("../..");
+
+exports["unknown elements should return HTMLUnknownElement"] = function (t) {
```

Comment:
Not sure, but there should be a list of all elements and their interfaces somewhere.